### PR TITLE
Add missing receptor vars to inventory appendix

### PR DESCRIPTION
**Pull request by michellemacrh**
Add missing receptor vars (receptor_listener_port, receptor_listener_protocol) to the Inventory file variables appendix

Affects: `titles/aap-containerized-install` and `titles/aap-installation-guide`

Containerized installation - Update receptor inventory file vars in Inventory file variables

https://issues.redhat.com/browse/AAP-35192

### CODE REVIEW

File: downstream/modules/platform/ref-ansible-inventory-variables.adoc [hunk 1]
```diff
@@ -11,47 +11,47 @@ For a list of global configuration options, see link:https://docs.ansible.com/an
...
-| *`ansible_password`* | The password to authenticate to the host.
+Default: 22 for SSH
+| `ansible_user` | The user name to use when connecting to the host.
+| `ansible_password` | The password to authenticate to the host.
 
 Never store this variable in plain text.
 
 Always use a vault.
-| *`ansible_ssh_private_key_file`* | Private key file used by SSH. Useful if using multiple keys and you do not want to use an SSH agent.
+| `ansible_ssh_private_key_file` | Private key file used by SSH. Useful if using multiple keys and you do not want to use an SSH agent.
```
Comment (ariordan-redhat) [inline downstream/modules/platform/ref-ansible-inventory-variables.adoc:30]:
```suggestion
| `ansible_ssh_private_key_file` | Private key file used by SSH. 
```
Duplicated sentence

File: downstream/modules/platform/ref-ansible-inventory-variables.adoc [hunk 2]
```diff
@@ -11,47 +11,47 @@ For a list of global configuration options, see link:https://docs.ansible.com/an
...
 
 This setting overrides the default behavior to use the system SSH.
-This can override the ssh_executable setting in `ansible.cfg`.
-| *`ansible_ssh_executable`* | Added in version 2.2.
+This can override the `ssh_executable` setting in `ansible.cfg`.
+| `ansible_ssh_executable` | Added in version 2.2.
 
-This setting overrides the default behavior to use the system SSH. This can override the ssh_executable setting in 'ansible.cfg'.
-| *`ansible_shell_type`* | The shell type of the target system.
+This setting overrides the default behavior to use the system SSH. This can override the `ssh_executable` setting in 'ansible.cfg'.
```
Comment (ariordan-redhat) [inline downstream/modules/platform/ref-ansible-inventory-variables.adoc:46]: ansible_ssh_executable is added twice

File: downstream/modules/platform/ref-ansible-inventory-variables.adoc (hunk 1 above)
Comment (michellemacrh) [reply downstream/modules/platform/ref-ansible-inventory-variables.adoc:30]: Good catch - it looks like this variable is listed twice !

**Review by ariordan-redhat — APPROVED** [review]
LGTM

**Pull request merged**